src/js/views/dom.js

```diff
@@ -121,19 +121,29 @@ export function createElement(el, className, children, id) {
 const handleWindowResize = () =>
   resizeIframe({ height: window.innerHeight, width: window.innerWidth })
 
+let listenerFunc
+
 function handleKeyPress(onClose) {
-  return function innerFunc(e) {
+  function innerFunc(e) {
     if (e.key === 'Escape') {
       e.preventDefault()
       onClose && onClose()
       closeModal()
-      window.removeEventListener('keydown', innerFunc)
     }
   }
+
+  listenerFunc = innerFunc
+
+  return listenerFunc
 }
 
 export function closeModal() {
   window.removeEventListener('resize', handleWindowResize)
+  if (listenerFunc) {
+    window.removeEventListener('keydown', listenerFunc)
+    listenerFunc = undefined
```

Comment:
thoughts on resetting this to null instead of undefined? 

I know some people think undefined should be reserved for values that are declared but not defined yet

Reply:
Yeah fair call, has been updated in a54e14e 👍